specification/frontdoor/resource-manager/Microsoft.Network/stable/2019-03-01/webapplicationfirewall.json

```diff
@@ -0,0 +1,813 @@
+        },
+        "ruleSetType": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Type of the managed rule set."
+        },
+        "ruleSetVersion": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Version of the managed rule set type."
+        },
+        "ruleGroups": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/ManagedRuleGroupDefinition"
+          },
+          "description": "Rule groups of the managed rule set."
+        }
+      }
+    },
+    "ManagedRuleGroupDefinition": {
+      "description": "Describes a managed rule group.",
+      "properties": {
+        "ruleGroupName": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Name of the managed rule group."
+        },
+        "description": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Description of the managed rule group."
+        },
+        "rules": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/ManagedRuleDefinition"
+          },
+          "description": "List of rules within the managed rule group."
+        }
+      }
+    },
+    "ManagedRuleDefinition": {
+      "description": "Describes a managed rule definition.",
+      "properties": {
+        "ruleId": {
+          "description": "Identifier for the managed rule.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "description": {
+          "description": "Describes the functionality of the managed rule.",
+          "readOnly": true,
+          "type": "string"
+        }
+      }
+    },
+    "ActionType": {
+      "description": "Defines the action to take on rule match.",
+      "type": "string",
+      "enum": [
+        "Allow",
+        "Block",
+        "Log",
+        "Redirect"
+      ],
+      "x-ms-enum": {
+        "name": "ActionType",
+        "modelAsString": true
+      }
+    },
+    "ErrorResponse": {
+      "description": "Error response indicates Front Door service is not able to process the incoming request. The reason is provided in the error message.",
+      "type": "object",
+      "properties": {
+        "code": {
+          "description": "Error code.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "message": {
+          "description": "Error message indicating why the operation failed.",
+          "readOnly": true,
+          "type": "string"
+        }
+      }
+    },
+    "FrontendEndpointLink": {
+      "description": "Defines the Resource ID for a Frontend Endpoint.",
+      "type": "object",
+      "readOnly": true,
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Resource ID."
+        }
+      }
+    },
+    "ProvisioningState":{
```

Comment:
As far as I understood, this is the new model in this API.

Reply:
Reverted the new model change since this change does not modify the API version.

---
In reply to: [285735596](https://github.com/Azure/azure-rest-api-specs/pull/5918#discussion_r285735596) [](ancestors = 285735596)